actor/v3action/space_test.go

```diff
@@ -145,4 +147,78 @@ var _ = Describe("Space", func() {
 			})
 		})
 	})
+
+	Describe("GetSpaceByNameAndOrganization", func() {
+		var (
+			spaceName string
+			orgGUID   string
+
+			space      Space
+			warnings   Warnings
+			executeErr error
+		)
+
+		BeforeEach(func() {
+			spaceName = "some-space"
+			orgGUID = "some-org-guid"
+		})
+
+		JustBeforeEach(func() {
+			space, warnings, executeErr = actor.GetSpaceByNameAndOrganization(spaceName, orgGUID)
+		})
+
+		Context("when the GetSpace call is successful", func() {
+			Context("when the cloud controller returns back one space", func() {
+				BeforeEach(func() {
+					fakeCloudControllerClient.GetSpacesReturns(
+						[]ccv3.Space{{GUID: "some-space-guid", Name: spaceName}},
+						ccv3.Warnings{"some-space-warning"}, nil)
+				})
+
+				It("returns back the first space and warnings", func() {
+					Expect(executeErr).ToNot(HaveOccurred())
+
+					Expect(space).To(Equal(Space{
+						GUID: "some-space-guid",
+						Name: spaceName,
+					}))
+					Expect(warnings).To(ConsistOf("some-space-warning"))
+
+					Expect(fakeCloudControllerClient.GetSpacesCallCount()).To(Equal(1))
+					Expect(fakeCloudControllerClient.GetSpacesArgsForCall(0)).To(Equal(url.Values{
+						ccv3.NameFilter:             []string{spaceName},
+						ccv3.OrganizationGUIDFilter: []string{orgGUID},
+					}))
+				})
+			})
+
+			Context("when the cloud controller returns back no spaces", func() {
+				BeforeEach(func() {
+					fakeCloudControllerClient.GetSpacesReturns(
+						nil, ccv3.Warnings{"some-space-warning"}, nil)
+				})
+
+				It("returns back the first space and warnings", func() {
```

Comment:
the spec should say returns a SpaceNotFoundError

Reply:
Whoops! Fixed :)